index.d.ts

```diff
@@ -95,7 +95,7 @@ declare class Sound {
    * @param onError Optional callback function if loading file failed
    * @param streamType streamType
    */
-  setStreamType(streamType: { audioStreamType: AudioManagerAudioStreamType }, onError: (error: any) => void)
```

Comment:
I think `setStreamType` will be needed only stream type.

Reply:
Just out of curiosity, do we have a plan to sync with upstream including this?

Reply:
Currently not. It needs more working to sync with upstream.

1) `setAudioStreamType` is deprecated in the API 26.
2) There have been some redundant codes in the `prepare()` when we synced with upstream before.

Let's handle on this other issue.